www/src/pages/components/chip/react/index.mdx

```diff
@@ -39,6 +39,22 @@ function FilterChipExample() {
 }
 ```
 
+With the `icon` prop:
+
+```jsx
+function FilterChipIconExample() {
+    const [isSelected, setIsSelected] = React.useState(true);
+    return (
+        <FilterChip
+            text="I'm Selected"
```

Comment:
Small suggestion. The initial state is not selected and it feels strange to see a label that says "I'm selected" but it's not. Maybe we can use a more generic label instead?

Reply:
Ah, meant to change this. Thanks for pointing it out. Done.